packages/flutter_tools/lib/src/commands/analyze_once.dart

```diff
@@ -48,17 +48,15 @@ class AnalyzeOnce extends AnalyzeBase {
     final String currentDirectory =
         (workingDirectory ?? fileSystem.currentDirectory).path;
 
-    // find directories from argResults.rest
-    final Set<String> directories = Set<String>.of(argResults.rest
+    // find directories or files from argResults.rest
```

Comment:
does the dartanalyzer support mixing and matching files with directories?

Reply:
Yeah seems like it support mixing I will also add a test for that case